no-jira: register NephronOptions and documentation tweaks (#49)
OpenNMS/nephron: docs/modules/configuration/pages/requirements.adoc

* Kafka server with forwarder enabled (see *Operation>Flow Support>Scale Flow Processing with Sentinel>Set up Message Broker: Kafka* in the core docs)
* A fast-caching DNS resolver if using reverse DNS

After Nephron set up, you must do the following:

 * xref:kafka.adoc#kafka-config[Configure Kafka]
 * xref:persistence.adoc#nephron-persistence[Configure persistence]
 * xref:monitor.adoc#nephron-monitor[Monitor Nephron]

[[flink]]
== Run on Flink
Once you set up your Flink cluster and size it to your flows volume, run the following to set up Nephron on Flink:

`./bin/flink run --parallelism 1 --class org.opennms.nephron.Nephron assemblies/flink/target/nephron-flink-bundled-*.jar --runner=FlinkRunner --jobName=nephron --checkpointingInterval=600000 --autoCommit=false`

> A list of all Nephron specific parameters can be output by executing the following command:

Reviewer: ```suggestion
Run the following command to output a list of all Nephron-specific parameters:
```

`java -jar assemblies/flink/target/nephron-flink-bundled-*.jar --help=NephronOptions`

> All Flink related parameters are output when executing:

Reviewer: ```suggestion
Run the following to output all Flink-related parameters:
```
